contracts/minting/MintController.sol

```diff
@@ -134,6 +141,37 @@ contract MintController is Controller {
         return internal_setMinterAllowance(minter, newAllowance);
     }
 
+    /**
+     * @dev decreases the minter allowance if and only if the minter is
+     * currently active. The controller can safely send a signed decrementMinterAllowance()
+     * transaction to a minter and not worry about it being used to undo a removeMinter()
+     * transaction.
+     */
+    function decrementMinterAllowance(
+        uint256 _allowanceDecrement
+    )
+    public
```

Comment:
Minor - lines 153–155 (`public` through `returns (bool)`) should be indented 4 spaces

Reply:
done!